gulp/GulpError.js

```diff
@@ -0,0 +1,11 @@
+class GulpError extends Error {
+    constructor(message, fatal = true) {
+        super();
+        this.message = message;
+        this.fatal = fatal;
```

Comment:
Currently unused - but later tasks will use this to defined whether a GulpError should be build-breaking or not.